### PR TITLE
add mouse absolute position

### PR DESCRIPTION
**Pull request by Palomino34**

### CODE REVIEW

File: GHIElectronics.TinyCLR.Devices.UsbClient/Mouse.cs [hunk 1]
```diff
@@ -75,6 +78,25 @@ public class Mouse : RawDevice {
         /// <summary>The current y coordinate of the cursor.</summary>
         public int CursorY => this.y;
 
+        /// <summary>The current wheel position.</summary>
```
Comment (gus-ghielec) [inline GHIElectronics.TinyCLR.Devices.UsbClient/Mouse.cs]: Is this comment correct?

File: GHIElectronics.TinyCLR.Devices.UsbClient/Mouse.cs [hunk 2]
```diff
@@ -287,43 +309,57 @@ public Mouse(UsbClientController usbClientController, UsbClientSetting usbClient
         /// <param name="stepY">The amount by which to increment the y position each request.</param>
         /// <param name="stepDelay">How long to wait between each step.</param>
         public void MoveCursorTo(int x, int y, int stepX, int stepY, int stepDelay) {
```
Comment (gus-ghielec) [inline GHIElectronics.TinyCLR.Devices.UsbClient/Mouse.cs]: Why not just use mouse.Move(x,y)

**Review by gus-ghielec — CHANGES_REQUESTED** [review]
Do full code review/test and make necessary improvements

File: GHIElectronics.TinyCLR.Devices.UsbClient/Mouse.cs (hunk 2 above)
Comment (Palomino34) [reply GHIElectronics.TinyCLR.Devices.UsbClient/Mouse.cs]: MoveTo or MoveCursorTo is more clear but I just changed as you asked.

**Pull request merged**